public/meetup.html

```diff
@@ -330,34 +315,17 @@ <h2 class="pl-4 text-2xl font-bold">O co chodzi?</h2>
           </div>
         </div>
         <div class="rounded-3xl border border-violetb p-8">
-          <div class="hidden flex-row lg:flex">
+          <div class="flex flex-row items-start lg:items-center">
             <img
               src="./images/icons/user.svg"
               class="size-9 self-start"
-              alt="Ikona użytkownika"
+              alt="Ikona żarówki"
             />
-
-            <div class="flex-col md:pl-8">
+            <div class="flex-col pl-4 md:pl-8">
               <h2 class="text-2xl font-bold">Dla kogo?</h2>
-              <p class="mt-5 max-h-[114px] text-base leading-5 tracking-tight">
-                Spotkania są dostępne dla każdego, bez względu na stopień
-                doświadczenia zawodowego czy wykorzystywaną technologię.
-                Zapraszamy programistów, grafików, administratorów, devopsów
-                oraz pasjonatów i hobbystów.
-              </p>
-            </div>
-          </div>
-          <div class="flex flex-col lg:hidden">
-            <div class="flex flex-row items-center">
-              <img
-                src="./images/icons/user.svg"
-                class="size-9 self-start"
-                alt="Ikona użytkownika"
-              />
-              <h2 class="pl-4 text-2xl font-bold">Dla kogo?</h2>
-            </div>
-            <div class="flex-col pt-4 md:pl-8">
-              <p class="text-sm font-normal leading-5 tracking-tight">
+              <p
+                class="-ml-12 mt-5 text-sm leading-5 tracking-tight lg:ml-0 lg:mt-0 lg:text-base"
+              >
```

Comment:
Can we get rid of these? One line is enough for simple tags like this.

Reply:
fixed